Error when init mm-snap

Error when running `mm-snap init`

```
MetaMask Snaps: Initialize

package name: (example) 
version: (1.0.0) 
description: 
entry point: (index.js) 
test command: 
git repository: 
keywords: 
author: 
license: (ISC) 
About to write to /home/vid/Documents/Blockchain_LabUM/MetaMask/example/package.json:

{
  "name": "example",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC"
}


Is this OK? (yes) yes

Init: Building 'snap.manifest.json'...

Use default Snap manifest values? (yes) yes
Using default values...

Init: Created 'snap.manifest.json'.
Init: Created 'index.js'.
Init: Created 'index.html'.
Init: Wrote 'snap.config.json' config file
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'babelify' from '/home/vid/Documents/Blockchain_LabUM/MetaMask/example'
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:146:35
    at processDirs (/home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:299:39)
    at isdir (/home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:306:32)
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:34:69
    at FSReqCallback.oncomplete (fs.js:169:21)
Emitted 'error' event on Browserify instance at:
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/browserify/index.js:375:34
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:148:17
    at processDirs (/home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:299:39)
    [... lines matching original stack trace ...]
    at FSReqCallback.oncomplete (fs.js:169:21) {
  code: 'MODULE_NOT_FOUND'
}
```


Version: 0.8.0
Installation: `yarn global add @metamask/snaps-cli`

--- a/packages/snaps-cli/src/cmds/build/bundle.test.ts
+++ b/packages/snaps-cli/src/cmds/build/bundle.test.ts
@@ -4,6 +4,7 @@ import { bundle } from './bundle';
 import * as bundleUtils from './bundleUtils';
 
 jest.mock('browserify');
+jest.mock('babelify', () => 'mockBabelify');
 
 describe('bundle', () => {
   global.snaps = {
@@ -40,7 +41,7 @@ describe('bundle', () => {
       expect(mockBrowserify).toHaveBeenCalledWith('src', { debug: true });
       expect(mockTransform).toHaveBeenCalledTimes(1);
       expect(mockTransform).toHaveBeenCalledWith(
-        'babelify',
+        'mockBabelify',
         expect.objectContaining({ global: true }),
       );
       expect(createStreamMock).toHaveBeenCalledTimes(1);
@@ -75,7 +76,7 @@ describe('bundle', () => {
       expect(mockBrowserify).toHaveBeenCalledWith('src', { debug: false });
       expect(mockTransform).toHaveBeenCalledTimes(1);
       expect(mockTransform).toHaveBeenCalledWith(
-        'babelify',
+        'mockBabelify',
         expect.objectContaining({ global: true }),
       );
       expect(createStreamMock).toHaveBeenCalledTimes(1);
@@ -110,7 +111,7 @@ describe('bundle', () => {
       expect(mockBrowserify).toHaveBeenCalledWith('src', { debug: true });
       expect(mockTransform).toHaveBeenCalledTimes(1);
       expect(mockTransform).toHaveBeenCalledWith(
-        'babelify',
+        'mockBabelify',
         expect.objectContaining({ global: false }),
       );
       expect(createStreamMock).toHaveBeenCalledTimes(1);

--- a/packages/snaps-cli/src/cmds/build/bundle.ts
+++ b/packages/snaps-cli/src/cmds/build/bundle.ts
@@ -4,7 +4,7 @@ import { TranspilationModes } from '../../builders';
 import { createBundleStream, closeBundleStream } from './bundleUtils';
 
 // We need to statically import all Browserify transforms and all Babel presets
-// presets and plugins, and calling `require` is the sanest way to do that.
+// and plugins, and calling `require` is the sanest way to do that.
 /* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, node/global-require */
 
 /**

--- a/packages/snaps-cli/src/cmds/build/bundle.ts
+++ b/packages/snaps-cli/src/cmds/build/bundle.ts
@@ -3,6 +3,10 @@ import { YargsArgs } from '../../types/yargs';
 import { TranspilationModes } from '../../builders';
 import { createBundleStream, closeBundleStream } from './bundleUtils';
 
+// We need to statically import all Browserify transforms and all Babel presets
+// presets and plugins, and calling `require` is the sanest way to do that.
+/* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, node/global-require */
+
 /**
  * Builds a Snap bundle JSON file from its JavaScript source.
  *
@@ -25,11 +29,11 @@ export function bundle(
     const bundler = browserify(src, { debug });
 
     if (transpilationMode !== TranspilationModes.none) {
-      bundler.transform('babelify', {
+      bundler.transform(require('babelify'), {
         global: transpilationMode === TranspilationModes.all,
         presets: [
           [
-            '@babel/preset-env',
+            require('@babel/preset-env'),
             {
               targets: {
                 browsers: ['chrome >= 66', 'firefox >= 68'],
@@ -38,11 +42,11 @@ export function bundle(
           ],
         ],
         plugins: [
-          '@babel/plugin-transform-runtime',
-          '@babel/plugin-proposal-class-properties',
-          '@babel/plugin-proposal-object-rest-spread',
-          '@babel/plugin-proposal-optional-chaining',
-          '@babel/plugin-proposal-nullish-coalescing-operator',
+          require('@babel/plugin-transform-runtime'),
+          require('@babel/plugin-proposal-class-properties'),
+          require('@babel/plugin-proposal-object-rest-spread'),
+          require('@babel/plugin-proposal-optional-chaining'),
+          require('@babel/plugin-proposal-nullish-coalescing-operator'),
         ],
       });
     }

